src/heroku-cli/devcontainer-feature.json

```diff
@@ -2,14 +2,14 @@
     "name": "Heroku CLI (via cli-assets.heroku.com)",
     "id": "heroku-cli",
     "version": "1.0.2",
```

Comment:
Please, bump the version to `1.0.3`. I should have spotted it earlier

Reply:
I added #70 to create a check for that in the future

Reply:
thanks! good catch